### ISSUE TITLE
'Edited' user panel appears above 'asked' user panel instead of to the left of it

### ISSUE THREAD

**Issue by AmperAndSand** [soscripted/sox#444]
**Describe the bug**
'Edited' user panel appears above 'asked' user panel instead of its default location (left of 'asked' user panel), which messes up the page's layout. The issue doesn't happen if SOX is disabled.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to https://gaming.stackexchange.com/questions/4999/how-can-i-tell-if-a-corpse-is-safe-to-eat or https://superuser.com/questions/149329/what-is-the-curl-command-line-syntax-to-do-a-post-request
2. Notice that the 'edited' user panel appears above the 'asked' user panel instead of to the left of it.

**SOX errors logged in console**
```
VM194:3 setNavigator - uri: https://gaming.stackexchange.com/questions/4999/how-can-i-tell-if-a-corpse-is-safe-to-eat
SOUP.user.js:2236 soup injecting fixes
VM200:31 soup window.jQuery setter initialized
VM200:31 soup init complete
core.js:43 [Nano] Nano Defender Activated :: gaming.stackexchange.com
rules-common.js:295 [Nano] Excluded :: uBO-Extra
VM200:31 soup jQuery init fixes applied
VM200:31 soup setup complete
how-can-i-tell-if-a-corpse-is-safe-to-eat:2221 GET https://secure.quantserve.com/quant.js net::ERR_BLOCKED_BY_CLIENT
(anonymous) @ how-can-i-tell-if-a-corpse-is-safe-to-eat:2221
(anonymous) @ how-can-i-tell-if-a-corpse-is-safe-to-eat:2229
how-can-i-tell-if-a-corpse-is-safe-to-eat:2227 GET https://sb.scorecardresearch.com/beacon.js net::ERR_BLOCKED_BY_CLIENT
(anonymous) @ how-can-i-tell-if-a-corpse-is-safe-to-eat:2227
(anonymous) @ how-can-i-tell-if-a-corpse-is-safe-to-eat:2229
index.html:68 SOX: Could not find user user link for: 
index.html:68 SOX: <div id=​"answer-34323" class=​"answer deleted-answer deleted-hidden" data-answerid=​"34323" itemprop=​"suggestedAnswer" itemscope itemtype=​"http:​/​/​schema.org/​Answer" data-votes=​"0" style=​"display:​ none;​">​…​</div>​
VM200:31 soup JS fixes applied
VM200:31 soup subscribed to realtime feed for question 4999 on site 41
```
**Screenshots/GIFs**
![](https://user-images.githubusercontent.com/1404745/66104707-0f1b4a00-e5ec-11e9-8cda-dc3ed37d3d8e.png)
![](https://user-images.githubusercontent.com/1404745/66104708-0f1b4a00-e5ec-11e9-87ff-543cd12df5ee.png)

**Environment**
SOX version: 2.6.6 DEV
Platform: Violentmonkey

Tested in Brave (Violentmonkey) 0.68.142 Chromium: 77.0.3865.90 (Official Build) (64-bit), Chrome (Tampermonkey) 77.0.3865.75 (Official Build) (64-bit) and Firefox (Tampermonkey) 69.0.1 (64-bit). Same SOX version above and settings.

---

### Features Enabled 

    ["Appearance-addAuthorNameToInboxNotifications","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-highlightQuestions","Appearance-isQuestionHot","Appearance-localTimestamps","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-metaNewQuestionAlert","Appearance-scrollToTop","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-unspoil","Appearance-addTimelineAndRevisionLinks","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-copyCommentsLink","Comments-showCommentScores","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-kbdAndBullets","Editing-titleEditDiff","Flags-flagOutcomeTime","Flags-flagPercentages","Sidebar-linkedToFrom","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-warnNotLoggedIn","Extras-dailyReviewBar"]

**Comment by shu8** [comment]
Thanks, I've reproduced this and will try getting it fixed ASAP :)

**Comment by shu8** [comment]
@AmperAndSand fixed in [dev 2.6.8](https://github.com/soscripted/sox/raw/dev/sox.user.js) :)
